src/components/BlogPage/Blogs.js

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+
+import { graphql, Link, useStaticQuery  } from "gatsby"
+
+const Blogs = () => {
+
+  const query = useStaticQuery(graphql`
+    {
+      allWpPost(sort: {order: DESC, fields: date}, skip: 1) {
+        edges {
+          node {
+            id
+            title
+            date(formatString: "YYYY-MM-DD")
+            slug
+          }
+        }
+      }
+    }
+  `)
+
+  const { edges } = query.allWpPost
+
+  console.log("edges", edges);
+
+  return (
+    <div>
```

Comment:
`{ edges.length > 0 && <div></div> }` can we do it something like this?

Reply:
Please, let me know about the new commit.